ts/screens/wallet/FirstTransactionSummaryScreen.tsx

```diff
@@ -0,0 +1,239 @@
+/**
+ * This screen shows the transaction details.
+ * It should occur after the transaction identification by qr scanner or manual procedure.
+ * TODO:
+ * - integrate contextual help
+ *    https://www.pivotaltracker.com/n/projects/2048617/stories/158108270
+ *
+ * - check availability of displayed data. Define optional data and implement their rendering as preferred
+ *
+ *
+ */
+
+import {
+  Body,
+  Button,
+  Container,
+  Content,
+  H1,
+  H3,
+  Icon,
+  Left,
+  Text,
+  View
+} from "native-base";
+import * as React from "react";
+import { Image, StyleSheet } from "react-native";
+import { Col, Grid, Row } from "react-native-easy-grid";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import { WalletAPI } from "../../api/wallet/wallet-api";
+import { WalletStyles } from "../../components/styles/wallet";
+import AppHeader from "../../components/ui/AppHeader";
+import PaymentSummaryComponent from "../../components/wallet/paymentSummary/PaymentSummaryComponent";
+import UpdatedPaymentSummaryComponent from "../../components/wallet/paymentSummary/UpdatedPaymentSummaryComponent";
+import I18n from "../../i18n";
+import variables from "../../theme/variables";
+import {
+  NotifiedTransaction,
+  TransactionEntity,
+  TransactionSubject
+} from "../../types/wallet";
+
+type Props = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+
+const styles = StyleSheet.create({
+  padded: {
+    paddingRight: variables.contentPadding,
+    paddingLeft: variables.contentPadding
+  }
+});
+
+const transactionDetails: Readonly<
+  NotifiedTransaction
+> = WalletAPI.getNotifiedTransaction();
+const entityDetails: Readonly<
+  TransactionEntity
+> = WalletAPI.getTransactionEntity();
+const subjectDetails: Readonly<
+  TransactionSubject
+> = WalletAPI.getTransactionSubject();
+
+export class FirstTransactionSummaryScreen extends React.Component<
+  Props,
+  never
+> {
+  constructor(props: Props) {
+    super(props);
+  }
+
+  private goBack() {
+    this.props.navigation.goBack();
+  }
+
+  /**
+   * Depending on the comparison between the amount on the notice and the amount saved remotely by the lender
+   * it will be displayed a different component. If the values differ, then the user can display both the values
+   * and a brief exmplanation related to the update.
+   */
+  private isAmountUpdated() {
+    return (
+      transactionDetails.currentAmount !== transactionDetails.notifiedAmount
+    );
+  }
+
+  private getSummary() {
+    if (this.isAmountUpdated() === false) {
+      return (
+        // if the amount had been updated, it will be displayed and presented with a brief description
+        <UpdatedPaymentSummaryComponent
```

Comment:
Based on the suggestion to merge `UpdatedPaymentSummaryComponent` and `PaymentSummaryComponent`, this `if ... else ...` will become something along the lines of:
```typescript
        <PaymentSummaryComponent
          navigation={this.props.navigation}
          amount={transactionDetails.notifiedAmount.toString()}
          updatedAmount={transactionDetails.currentAmount.toString()}
          expireDate={transactionDetails.expireDate.toLocaleDateString()}
          tranche={transactionDetails.tranche} />
```
And `PaymentSummaryComponent` will handle `amount` and `updatedAmount` to decide what to render.